woocommerce/woocommerce-gutenberg-products-block
# Add margin support on the Product Image block

This PR adds support for margin customization on the Global Style.


Fixes #5671 

### Screenshots



### Testing
### Manual Testing

How to test the changes in this Pull Request:



This PR adds support for global style for the `Product Image` block.

Now, the user can edit the style for:
- margin



#5671 

### Screenshots

![image](https://user-images.githubusercontent.com/4463174/155329953-318d3348-c112-4e5f-95cf-a35757498690.png)
*without global style*

![image](https://user-images.githubusercontent.com/4463174/155332770-ad07f00d-ac9f-41e6-a8b1-21073b44d7d1.png)
*with global style*

### Testing

### Manual Testing

Check out this branch:

1. Use `WordPress 5.9`.
2. Install and enable the `Twenty Twenty-Two` theme.
3. Add the `All Products` block  (this block contains `Product Image`) to a post.
4. Get the focus on the `Product Image` block.
5. On the right sidebar, personalize the margin style.
6. Go on the page and check if there are changes.
7. Reset to default using the `Reset` button from the different sections.
8. On the Editor page click on the `Styles` icon on the right-top corner.
9. Verify that the `Product Image Block` is shown under the `Blocks` section. Personalize again the margin style.
10. Save your changes.
11. Go on the page created earlier and check if all styles are applied correctly.
12. Edit your previous post/page again.
13. Change again the styles.
14. Save your changes.
15. Check if these styles have priority over the styles from the Site Editor.
